README.md

```diff
@@ -1,45 +1,102 @@
 
 ## HIP Work Flow
-Parties involved in this process are you, the HIP author, the HIP editors, and the Hashgraph Developers.
 
-Evaluate your idea: Before you begin, ask the Hashgraph community first if your idea is original to save time. Make sure the idea is applicable to the entire community and not to one particular author. While it may be a good idea for an author it may not be the case for all areas of the Hashgraph community. An excellent place to discuss your proposal is here in the Issues section of this repository. You can start creating a more formalized language around your HIP.
+Parties involved in this process are you (the HIP author), the HIP editors, and the Hedera Governing Council (Hedera Hashgraph LLC). 
+
+#### The following is the process that a successful HIP will move along: 
+
+   - Work in Progress (WIP) 
+
+   - Draft 
+
+   - Last Call 
+
+   - Accepted by the community 
+
+   - Proposed to the Hedera Governing Council 
+
+   - Accepted by the Hedera Governing Council 
+
+   - Final 
 
-Following is the process that a successful EIP will move along:
+[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED BY THE COMMUNITY ] -> [ PROPOSED TO HGC ] -> [ ACCEPTED BY HGC ] -> [ FINAL ] 
 
-[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+HGC = [Hedera Governing Council](https://hedera.com/council)
 
-The status change of each HIP is requested by the HIP author and it is to be reviewed by the HIP editors. To update the status, use a pull request. For convince, include a link for people to continue discussing your HIP. HIP editors should follow the following guidelines when reviewing requests:
-- Active
-- Work in Progress (WIP)
-- Draft
-- Last Call
-- Accepted 
-- Final
+Alternative statuses include: 
 
-Exceptional statuses:
-- Deferred: A core HIPs that have been put off for a future hard fork. 
-- Rejected: An HIP that is fundamentally broken or a core HIP that was rejected by the Core Devs.
-- Superseded: An HIP which was previously final but is no longer considered state-of-the-art.
+   - Deferred: HIPs with this status has been put off to be discussed again at a later date. 
+
+   - Rejected: HIPs with this status are either: fundamentally broken, not accepted by the community, or not accepted by the Hedera Governing Council. 
+
+   - Superseded: HIPs with this status were previously at some stage of acceptance, but is no longer considered state-of-the-art. 
+
+The status change of each HIP is requested by the HIP author and it is to be reviewed by the HIP editors. To update the status, use a pull request. 
+
+For convenience, always include a link for people to continue discussing your HIP or see the conversations that have already taken place. 
+
+## Before filling your HIP
+
+#### Step 1)
```

Comment:
I think a numbered list is more appropriate instead of headers with step numbers